ChangeLog.d/add-milliseconds-time-api.txt

```diff
@@ -0,0 +1,7 @@
+API changes
+   * Add `mbedtls_ms_time` function, `mbedtls_ms_time_t` type to get
+     milliseconds. That is required for TLS 1.3 ticket lifetime. And
+     `mbedtls_ms_time` allows alternative implementation with
+     `MBEDTLS_PLATFORM_MS_TIME_ALT`, `MBEDTLS_PLATFORM_MS_TIME_TYPE_MACRO` and
+     `MBEDTLS_PRINTF_MS_TIME`.
```

Comment:
I'd probably mention the new type before the new function - and I'd use `()` on the function name. Probably not needed to add so much detail on the ALT interface.

```suggestion
   * Add new millisecond time type `mbedtls_ms_time_t` and `mbedtls_ms_time()`
     function, needed for TLS 1.3 ticket lifetimes. Alternative implementations
     can be created using an ALT interface.
```